### PR TITLE
Added new call to api for competencies per band, added new page to di…

### PR DESCRIPTION
**Pull request by Ryan-Jones1999**
…splay and added link from view roles page on the bandID; tests have also been added to test new service

### CODE REVIEW

File: app/views/viewroles.html [hunk 1]
```diff
@@ -23,6 +24,7 @@
       <th scope="row" class="govuk-table__header"><a href="{{role.specification}}">{{role.jobName}}</a></th>
       <td class="govuk-table__header">{{role.capabilityName}}</td>
       <td class="govuk-table__header govuk-!-width-one-half">{{role.specSummary}}</td>
+      <td class="govuk-table__header"><a href="viewcompetencies/{{role.bandLevelID}}">{{role.bandLevelID}}</a></td>
```
Comment (hellofadude) [inline app/views/viewroles.html:27]: We should be displaying the band level name not the Id

File: app/controller/employeeController.js [hunk 2]
```diff
@@ -9,4 +9,11 @@ router.get('/viewroles',  async (req, res) =>  {
     res.render('viewroles', { roles: s })
 })
 
+router.get('/viewcompetencies/:BandID',  async (req, res) =>  {
+    var bandID = req.params.BandID
```
Comment (hellofadude) [inline app/controller/employeeController.js:13]: Please use ECMA - 6 i.e `const` or `let`

File: app/controller/employeeController.js [hunk 3]
```diff
@@ -9,4 +9,11 @@ router.get('/viewroles',  async (req, res) =>  {
     res.render('viewroles', { roles: s })
 })
 
+router.get('/viewcompetencies/:BandID',  async (req, res) =>  {
```
Comment (hellofadude) [inline app/controller/employeeController.js:12]: Please change filename to JobController

File: test/Unit/Service/employeeservicetest.js [hunk 4]
```diff
@@ -3,19 +3,28 @@ const MockAdapter = require('axios-mock-adapter');
 const chai = require("chai");
```
Comment (hellofadude) [inline test/Unit/Service/employeeservicetest.js:3]: Please change filename to `JobController` and rename tests accordingly

File: app/service/employee.js [hunk 5]
```diff
@@ -1,12 +1,22 @@
 const axios = require('axios');
-
```
Comment (hellofadude) [inline app/service/employee.js:2]: Change file name to JobController

File: test/Unit/Service/employeeservicetest.js [hunk 6]
```diff
@@ -25,12 +34,38 @@ describe('EmployeeService', function () {
...
             var error = await EmployeeService.viewjobroles();
             
             expect(error.message).to.equal('Failed to get job roles')
           })
+})
+describe('getCompetenciesPerBandLevel', function () {
+  it('should return competencies for a valid band', async () => {
+    var mock = new MockAdapter(axios);
+
+    const data = competencies;
```
Comment (hellofadude) [inline test/Unit/Service/employeeservicetest.js:48]: data is redundant

**Review by hellofadude — APPROVED** [review]

**Review by MihneaBocaKainos — APPROVED** [review]

**Pull request merged**